src/gui/owncloudgui.cpp

```diff
@@ -552,6 +558,11 @@ void ownCloudGui::setupContextMenu()
     }
 }
 
+void ownCloudGui::setupContextMenuIfVisible()
+{
+    if (_contextMenu && _contextMenu->isVisible())
```

Comment:
@mitya57 Note that there was significant follow up work done on context menu issues, at least 98efb075357fa64f424070e723db1786faa89d25, but possibly more.